### ISSUE TITLE
"e9patch", "revng", "dyninst" results and fixes - [closed]

### ISSUE THREAD

**Issue by ruricolist** [GrammaTech/lifter-eval#173]
***In GitLab by @vfoltsgt on May 24, 2022, 04:19 UTC:***

_Merges vfolts/ddisasm -> main_

Functional tests depend on a sandbox state and cannot be run in parallel.

*Migrated from GitLab: https://gitlab.com/GrammaTech/lifter-eval/-/merge_requests/101*

**Comment by ruricolist** [comment]
***In GitLab by @vfoltsgt on May 24, 2022, 04:36 UTC:***

added 1 commit

<ul><li>ac2a096b - &quot;dyninst&quot; results for x86 binaries</li></ul>

[Compare with previous version](/GrammaTech/lifter-eval/-/merge_requests/101/diffs?diff_id=400777012&start_sha=b3b4d02bc9ea85764dda3e6f4cec02bd90ddd544)

**Comment by ruricolist** [comment]
***In GitLab by @vfoltsgt on May 28, 2022, 01:23 UTC:***

Closing as superseded by !102.
